### PR TITLE
Property to tell to Tomcat to map JARs in WEB-INF/lib or WEB-INF/classes

### PR DESCRIPTION
**Pull request by RoiSoleil**
Use case : Chapter Ordering in https://tomcat.apache.org/tomcat-8.0-doc/config/resources.html

`Since both resources are PostResources, it might be expected that D:\Projects\external\classes will be searched for classes before D:\Projects\lib\library1.jar. However, by adding the JAR using a FileResourceSet, the JAR is mapped to /WEB-INF/lib and will be processed at application start along with the other JARs in /WEB-INF/lib. The classes from the JAR file will be added to the ClassResources which means they will be searched before the classes from D:\Projects\external\classes. If the desired behaviour is that D:\Projects\external\classes is searched before D:\Projects\lib\library1.jar then a slightly different configuration is required:

<Resources>
  <PostResources base="D:\Projects\external\classes"
                 className="org.apache.catalina.webresources.DirResourceSet"
                 webAppMount="/WEB-INF/classes"/>
  <PostResources base="D:\Projects\lib\library1.jar"
                 className="org.apache.catalina.webresources.JarResourceSet"
                 webAppMount="/WEB-INF/classes"/>
</Resources>
In short, the JAR file should be added as a JarResourceSet mapped to /WEB-INF/classes rather than using a FileResourceSet mapped to /WEB-INF/lib.`

### CODE REVIEW

**Comment by alitokmen** [comment]
Thank you for the implementation, after accepting the PR I'll do a couple changes, namely:

- Adding the new property in the capabilities
- Have a default value for the property 

I'll also create the associated JIRA so we know this is coming with version 1.10.11.

**Comment by RoiSoleil** [comment]
Thx for the really quick merge !!!! Do you know when the next release will be ?

**Comment by alitokmen** [comment]
I can release "anytime", I'm just waiting for the PR https://github.com/codehaus-cargo/cargo/pull/423 to be revised

How quickly would you need this change?
